From david-b@pacbell.net  Thu Jun  7 16:12:31 2007
From: Tony Lindgren <tony@atomide.com>
Date: Tue, 29 May 2007 09:57:41 -0700
Subject: USB: Disable file_storage USB_CONFIG_ATT_WAKEUP
To: Greg KH <greg@kroah.com>
Cc: linux-usb-devel@lists.sourceforge.net, Tony Lindgren <tony@atomide.com>, Felipe Balbi <felipe.lima@indt.org.br>
Message-ID: <200705290957.42208.david-b@pacbell.net>
Content-Disposition: inline


From: Tony Lindgren <tony@atomide.com>

Disable file_storage USB_CONFIG_ATT_WAKEUP as it requires
user interaction during Chapter 9 tests.

Signed-off-by: Tony Lindgren <tony@atomide.com
Signed-off-by: Felipe Balbi <felipe.lima@indt.org.br>
Acked-by: Alan Stern <stern@rowland.harvard.edu>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/file_storage.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/usb/gadget/file_storage.c
+++ b/drivers/usb/gadget/file_storage.c
@@ -3963,8 +3963,7 @@ static int __init fsg_bind(struct usb_ga
 #endif
 
 	if (gadget->is_otg) {
-		otg_desc.bmAttributes |= USB_OTG_HNP,
-		config_desc.bmAttributes |= USB_CONFIG_ATT_WAKEUP;
+		otg_desc.bmAttributes |= USB_OTG_HNP;
 	}
 
 	rc = -ENOMEM;
